gulpfile.mjs

```diff
@@ -63,18 +50,24 @@ gulp.task('compile', gulp.series(
 gulp.task('dev', gulp.series(
   clean,
   'compile',
-  'watch',
+  watch,
   npmScriptTask('serve', ['--silent'])
 ))
 
+/**
+ * Watch task
+ * Runs tasks when files are modified
+ */
+gulp.task('watch', watch)
```

Comment:
I've added this so `gulp watch` still works, do we want this?

Reply:
I say nah. Can't think of a good enough reason to use it outside of the dev task.

Reply:
Thanks @domoscargin I'll remove it, rebase, and get this merged 😊